Part 2 - Landing Page/js/app.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * Manipulating the DOM exercise.
+ * Exercise programmatically builds navigation,
+ * scrolls to anchors from navigation,
+ * and highlights section in viewport upon scrolling.
+*/
+
+const ul = document.querySelector('#navbar__list');
+const sections = document.querySelectorAll('section');
+
+// build nav links
+const links = [
+    { id: '#section1', text: 'Section 1'},
+    { id: '#section2', text: 'Section 2'},
+    { id: '#section3', text: 'Section 3'},   
+];
```

Comment:
1. In rubrics, it requires 4 sections. though it doesn't differ much, but check rubrics should be on check list.
2. can consider reducing hard-coded repetition by loops. same for below anchors and sects.

Reply:
@sebz996 please resolve this with @leemoispace on a call

Reply:
Thanks for the helpful suggestion. After consulting with Martian, I will leave this code as is. But I've made a future note for such instances. 👍🏻